src/components/views/elements/Tooltip.tsx

```diff
@@ -87,7 +87,9 @@ export default class Tooltip extends React.Component<ITooltipProps> {
     }
 
     public componentDidUpdate() {
-        this.renderTooltip();
+        if (this.props.visible) {
+            this.renderTooltip();
```

Comment:
renderTooltip is also called on every scroll event even if not visible, I think the hiding of the tooltip should happen on a componentDidUpdate transition from visible->!visible and not be checked every renderTooltip, then renderTooltip just bails if `this.props.visible` - what do you think?